policy/modules/services/asterisk.te

```diff
@@ -110,7 +110,7 @@ corenet_udp_bind_sip_port(asterisk_t)
 corenet_sendrecv_generic_server_packets(asterisk_t)
 corenet_tcp_bind_generic_port(asterisk_t)
 corenet_udp_bind_generic_port(asterisk_t)
-corenet_dontaudit_udp_bind_all_ports(asterisk_t)
+corenet_udp_bind_all_unreserved_ports(asterisk_t)
```

Comment:
I think we want to keep the original line, so if it tries to bind to one of the defined ports, it will remain silent on audits.